app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -89,6 +97,44 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
         Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
     }
 
+    override fun showBenchmarkLoading() {
+        benchmarkButton.visibility = View.GONE
+        benchmarkErrorState.visibility = View.GONE
+        benchmarkLoading.visibility = View.VISIBLE
+    }
+
+    override fun showBenchmarkResult(result: BenchmarkResult) {
+        benchmarkButton.visibility = View.GONE
+        benchmarkLoading.visibility = View.GONE
+        benchmarkErrorState.visibility = View.GONE
+
+        result.benchmarks.forEach { (_, name, classifications) ->
+
+            val chips = ArrayList<FeedbackChip>()
+            classifications.mapTo(chips, { FeedbackChip(it) })
+
+            val predictionView = PredictionsView(this)
```

Comment:
Hmm... I see. Technically, I wouldn't say it's a prediction, right? For me, a prediction is more something like: "given these previous inputs, these are the chances a future result will fall into this category". For our case, it sounds more like classification: "given an image, I'll try to tell you what it is".

What's your opinion? (i'm not totally confident about what I just said lol)